src/universal/redux/storageMerger.js

```diff
@@ -9,26 +7,22 @@ export default function merger(initialState, persistedState) {
     const reducerName = persistedStateKeys[i];
     if (!whitelist.has(reducerName)) continue;
     const value = persistedState[reducerName];
-    if (reducerName === 'auth' && value) {
-      // Always perform a fresh decode for security's sake:
-      let authTokenObj = null;
-      try {
-        authTokenObj = jwtDecode(value.token);
-      } catch (e) {
-        console.warn(`unable to decode jwt: ${e}`);
-        continue;
-      }
-      if (authTokenObj.exp < Date.now() / 1000) {
+    if (reducerName === 'auth' && value && value.obj) {
+      if (value.obj.exp < Date.now() / 1000) {
         continue;
       }
     }
     if (reducerName === 'cashay') {
-      if (value && value.entities) {
-        // cashay now supports rehydrating sub data
+      if (value) {
+        const {User} = value.entities || {};
+        value.entities = User ? {User} : {};
+        value.result = {};
+
+        // cashay now supports rehydrating sub data, but this can't account for removed items
         // remove items with unbounded growth in production
-        value.result.presence = undefined;
-        value.entities.Presence = undefined;
-        value.entities.Task = undefined;
+        // value.result.presence = undefined;
```

Comment:
man, i'm so confused with the utility of persisted docs.
Any `result` from a subscription can't be trusted when the connection is lost because during that lost connection, that document could have been removed.(see #166 for how to fix this in the future).
When we get a `remove` from the server, we can't remove it from the client, because rethinkDB could send us an `ADD` (to another channel) and then a `REMOVE` (from the old channel). That means entities have unbounded growth. to mitigate this, we need TTL for docs, but since we don't have that right now, i just nuke all entities (except for `User` because cmon, how many people could you possibly log in as? Plus, that's how we get the name, picture, etc.)